File: official/docs/csharp/batches/CreateBatch.cs
using EasyPost;

EasyPost.ClientManager.SetCurrent("EASYPOST_API_KEY");

Batch batch = await Batch.Create(new Dictionary<string, object>() {
  { "shipments", new List<Dictionary<string, object>>() {
    new Dictionary<string, object>() 
      { "id", "shp_..." },
      {"id", "shp_..."} 
  } }
});

Console.Write(batch);

File: official/docs/php/batches/CreateBatch.php
<?php

\EasyPost\EasyPost::setApiKey('EASYPOST_API_KEY');

$batch = \EasyPost\Batch::create([
    'shipments' => [
        ['id' => 'shp_...'],
        ['id' => 'shp_...']
    ]
]);

echo $batch;
